packages/test-utils/src/reporter.ts

```diff
@@ -187,6 +188,28 @@ export class ReporterError extends Error {
   }
 }
 
+function getFallbackRunTitle() {
+  // for CI runs we don't want to set an explicit title
+  // dashboard is meant to use the commit/PR information retrieved on CI
+  if (process.env.CI) {
```

Comment:
we could use `ci-info` package to improve this check. `CI` should be set by GitHub Actions and a couple of other vendors though